### PR TITLE
1907-11-04 Page 1

### PR DESCRIPTION
**Pull request by WillHesse**
First Submittal

### CODE REVIEW

File: 1907-11-04.xml [hunk 1]
```diff
@@ -0,0 +1,1045 @@
...
+                    Worms &amp; Co.</p>
+                <p>Agents at PORT SAID, for the London and Queensland Line, Messrs. Wills &amp; Co.,
+                    Limited.</p>
+                <p>Messrs. Thos. Cook &amp; Son and the Anglo-American Hotel &amp; Steamer Company,
+                    CAIRO &amp; ALEXANDRIA.</p>
+                <p>For further particulars. Freight and Passage apply to G. BEYTS &amp; Co. Agents,
+                    Suez. <measure type="indexNo">31-12-906</measure></p>
+            </div>
+            
+            <cb n="2"/>
```
Comment (whanley) [inline 1907-11-04.xml]: Don't add cb here--only use it where a new column starts.

**Comment by whanley** [comment]
Any updates?

**Comment by WillHesse** [comment]
I’m finishing up deliverable one, I had a ton of issues with tesseract and I’m having some trouble with the blog post, I’m working on it and will have all of it done by Halloween. After doing deliverable one I believe it will be easier to successfully complete the other deliverables. I’m sorry, this is a huge learning curve for me, I’ve never really done a coding class. Thank you for being understanding though, every-time we talk I feel better about the assignments.
Thanks,
-William Hesse

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by whanley** [comment]
Great! Don't hesitate to ask for help when you need it.

**Comment by huoyanzhizhou** [comment]
Any news?

**Comment by whanley** [comment]
Will--solid work here but some formatting to be addressed. Come in to a work session and we'll get it sorted out.

**Comment by whanley** [comment]
A lot of this stuff is in the wrong repository. I'll take the pages images and other stuff and put it in the right places.

**Pull request merged**